Formula/c/chdig.rb

```diff
@@ -0,0 +1,22 @@
+class Chdig < Formula
+  desc "Dig into ClickHouse with TUI interface"
+  homepage "https://github.com/azat/chdig"
+  url "https://github.com/azat/chdig/archive/refs/tags/v25.3.1.tar.gz"
+  sha256 "69fd0063279d4299d760a43a039ceb70064c0e8f24c9bec57c3924aaadcd4216"
+  license "MIT"
+  head "https://github.com/azat/chdig.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+
+    generate_completions_from_executable(bin/"chdig", "--completions")
```

Comment:
```suggestion
    generate_completions_from_executable(bin/"chdig", "--completion", shells: [:bash, :zsh])
```

Reply:
yeah, I saw it does support fish as well?

Reply:
It does

```
      --completion <COMPLETION>
          [possible values: bash, elvish, fish, powershell, zsh]
```